InferenceSystem/README.md

```diff
@@ -68,13 +73,35 @@ export AZURE_COSMOSDB_PRIMARY_KEY="<yourprimarykey>"
 
 ```
 cd InferenceSystem
-python LiveInferenceOrchestrator.py
+python LiveInferenceOrchestrator.py --config ./config/Test/FastAI_LiveHLS_OrcasoundLab.yml
```

Comment:
Hm, currently `LiveInferenceOrchestrator.py` is inside `src` dir, one level deeper, so this should be `python src/LiveInferenceOrchestrator.py ...`? Haven't tested, something else might also break inside the orchestrator?

Reply:
Yep, you're right good catch. I locally tested as src/LiveInferenceOrchestrator.py but forgot to update.